app/views/results/common/_context_menu.js.erb

```diff
@@ -0,0 +1,121 @@
+<script>
+  /*
+    This file contains the Javascript code necessary for the functionality
+    of the annotation context menu. For the annotation context menu to
+    function properly, these dependencies are necessary:
+
+    Context-Menu and library related:
+      * jquery.ui-contextmenu.min.js (..\app\assets\javascripts\)
+        * Depends on jQuery and jQuery-ui
+      * context_menu.css (..\app\assets\stylesheets\)
+
+    Context-Menu button functionality:
+      * _annotations.js.erb (..\app\views\results\common\)
+
+     */
+  (function() {
+    var menu_items = {
+      new_annotation: {
+        title: 'New Annotation <kbd>Ctrl+?</kbd>',
+        cmd: 'new_annotation',
+        action: make_new_annotation,
+        disabled: true
+      },
+      pre_canned_annotations: {
+        title: 'Pre-canned <kbd>Ctrl+? <b>></b></kbd>',
```

Comment:
I suggest removing all keyboard shortcut hints from this PR. We'll have you add those in (and the shortcuts themselves) in a future task.